blueflood-http/src/main/java/com/rackspacecloud/blueflood/http/HttpResponder.java

```diff
@@ -50,18 +47,19 @@ public static void respond(ChannelHandlerContext ctx, FullHttpRequest req, FullH
             res.headers().add("Access-Control-Allow-Origin", CORS_ALLOWED_ORIGINS);
         }
 
-
         if (res.content() != null) {
-            setContentLength(res, res.content().readableBytes());
+            HttpUtil.setContentLength(res, res.content().readableBytes());
         }
 
-        if ( isKeepAlive(req) ) {
+        boolean isKeepAlive = HttpUtil.isKeepAlive(req);
+        if (isKeepAlive) {
             res.headers().add(CONNECTION, KEEP_ALIVE);
         }
 
         // Send the response and close the connection if necessary.
         ctx.channel().write(res);
-        if (req == null || !isKeepAlive(req)) {
+        if (req == null || !isKeepAlive) {
+            log.debug("Closing channel. isKeepAlive:" + isKeepAlive);
```

Comment:
Is there some specific information you have in mind? I can add the below line to the log.
```
ctx.channel().isActive()
```